related.tex

```diff
@@ -1,36 +1,32 @@
-the incrementalization process is repeated for the delta query.
+Incremental view maintenance~\cite{gupta-sigmod93, griffin-sigmod95,
+  chaudhuri-icde95, gupta-idb95, chirkova-book12} is a much studied
+problem in databases.  A survey of results for Datalog queries is
+present in~\cite{motik-ai19}.  DBToaster introduced recursive
+recursive IVM~\cite{ahmad-vldb09, koch-pods10}, where the
+incrementalization process is repeated for the delta query.
 
-Many custom algorithms were published for various classes of queries: e.g.~\cite{koch-pods16}
-handles positive nested relational calculus.  DYN~\cite{idris-sigmod17}
-and IDYN~\cite{idris-vldb18, idris-sigmod19} focus on acyclic conjunctive queries.  Instead
-of keeping the output view materialized they build data structures that allow efficiently
-querying the output views.  PAI maps~\cite{abeysinghe-sigmod22} are specially
-designed for queries with correlated aggregations.
+Many custom algorithms were published for various classes of queries:
+e.g.~\cite{koch-pods16} handles positive nested relational calculus.
+DYN~\cite{idris-sigmod17} and IDYN~\cite{idris-vldb18, idris-sigmod19}
+focus on acyclic conjunctive queries.  Instead of keeping the output
+view materialized they build data structures that allow efficiently
+querying the output views.  PAI maps~\cite{abeysinghe-sigmod22} are
+specially designed for queries with correlated aggregations.
 AJU~\cite{wang-sigmod20} focuses on foreign-key joins.  It is a matter
-of future work to evaluate whether custom \dbsp operators
-can match the efficiency of systems specialized for narrow classes
-of queries.
+of future work to evaluate whether custom \dbsp operators can match
+the efficiency of systems specialized for narrow classes of queries.
 
-\dbsp is a bottom-up system, which always produces eagerly
-the \emph{changes} to the output views.
-Instead of maintaining the output view entirely, \dbsp proposes
-generating deltas as the output of the computation (similar to the kSQL~\cite{jafarpour-edbt19}
-\texttt{EMIT CHANGES} queries).  The idea that both
-inputs and outputs to an IVM system are streams of changes
-seems trivial, but this is key to the symmetry of our solution:
-both in our definition of IVM~(\ref{def:inc}), and the fundamental
-reason that the chain rule exists --- the chain rule is the one that makes our
-structural induction IVM algorithm possible.
+\dbsp is a bottom-up system, which always produces eagerly the
+\emph{changes} to the output views.  Instead of maintaining the output
+view entirely, \dbsp proposes generating deltas as the output of the
+computation (similar to the kSQL~\cite{jafarpour-edbt19} \texttt{EMIT
+  CHANGES} queries).  The idea that both inputs and outputs to an IVM
+system are streams of changes seems trivial, but this is key to the
+symmetry of our solution: both in our definition of IVM, and the
+fundamental reason that the chain rule exists --- the chain rule is
```

Comment:
This wording seems a bit awkward. I don't have a suggestion for a fix, since I'm not sure what you want to say here.